subsys/bluetooth/mesh/Kconfig

```diff
@@ -316,6 +323,28 @@ config BT_MESH_LOOPBACK_BUFS
 	  The number of buffers allocated for the network loopback mechanism.
 	  Loopback is used when the device sends messages to itself.
 
+config BT_MESH_TX_SEG_RETRANS_CNT
+	int "Transport message segment retransmit attempts"
+	default 4
+	range 1 8
+	help
+	  Maximum number of transport message segment retransmit attempts
+	  for outgoing segment message.
+
+config BT_MESH_TX_SEG_RETRANS_TIMEOUT_UNICAST
+	int "Transport message segment retransmit interval for unicast messages"
+	default 400
+	range 200 500
+	help
+	  Maximum timeout value of retransmit segment message to unicast address.
```

Comment:
A timeout is the "Maximum time", but a "maximum timeout" would make sense if the timeout was negotiable.
Since it is the actual timeout I would drop the word "maximum" here.